pkg/mcs/scheduling/server/server.go

```diff
@@ -419,6 +466,18 @@ func (s *Server) startServer() (err error) {
 		return err
 	}
 
+	s.encryptionKeyManager, err = encryption.NewManager(s.GetClient(), &s.cfg.Security.Encryption)
+	if err != nil {
+		return err
+	}
+
+	h := handler.NewHandler(&svr{s})
+	s.hotRegionStorage, err = storage.NewHotRegionsStorage(
```

Comment:
Will it make the scheduling server stateful？

Reply:
It will create a leveldb in scheduling server. 
If hope to store it in api server, it means that we need to pull all hot region info from scheduling server per 10 minutes
https://docs.pingcap.com/zh/tidb/stable/information-schema-tidb-hot-regions-history

Reply:
Scheduling server should be stateless.